client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -284,9 +296,10 @@ const PluginsBrowser = ( { trackPageViews = true, category, search, searchTitle,
 				searchTerms={ [ 'seo', 'pay', 'booking', 'ecommerce', 'newsletter' ] }
 			/>
 
-			{ ! search && <Categories selected={ category } /> }
+			{ ! search && <Categories clearSearch={ clearSearch } selected={ category } /> }
```

Comment:
Categories component doesn't actually need to clear selected, it just needs to set a new selected. (Kind of want to remove state from this component at some point too.)